packages/eslint-plugin-tslint/src/rules/config.ts

```diff
@@ -63,7 +64,7 @@ export default createRule<Options, MessageIds>({
       description:
         'Wraps a TSLint configuration and lints the whole source using TSLint',
       // one off special category for this plugin
-      category: 'TSLint' as any, // eslint-disable-line @typescript-eslint/no-explicit-any
+      category: 'TSLint' as any, // eslint-disable-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-explicit-any
```

Comment:
probably a better solution?
```suggestion
      // @ts-expect-error
      category: 'TSLint',
```